server/controllers/finance/exchange.js

```diff
@@ -16,13 +16,14 @@ exports.list = function list(req, res, next) {
       enterprise = req.session.enterprise;
   
   sql =
-    'SELECT currency_id, rate, date ' +
+    'SELECT id, currency_id, rate, date ' +
     'FROM exchange_rate ' + 
     'WHERE enterprise_id = ? ' +
     'ORDER BY date;';
 
   db.exec(sql, [ enterprise.currency_id ])
```

Comment:
@jniles  It really enterprise.currency_id to be used in the query, because exports.list must return the list of exchange rates relative to the enterprise currency, the currency of the Company is obtained  through ' req.session.enterprise '